server/accounts.go

```diff
@@ -484,6 +484,20 @@ func (a *Account) TotalSubs() int {
 	return int(a.sl.Count())
 }
 
+// SubscriptionInterest returns true if this account has a matching subscription
+// for the given `subject`.
+func (a *Account) SubscriptionInterest(subject string) bool {
```

Comment:
Do we want to support wildcards here?

Reply:
I don't know. As you can see, this is calling sl.Match() which works for literals. We could make it call subsetxx but not sure if we need that. Clearly name of the API should not be misleading, or at very least add to the doc of that function.
Planning on rebasing from master and re-run anyway to see if we get new flappers.

Reply:
Add in docs and maybe TODO, then LGTM.